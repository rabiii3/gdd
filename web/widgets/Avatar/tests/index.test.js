import React from 'react';
import { mount } from 'enzyme';

jest.mock('../../../../lib/utils', () => ({
  getColor: () => 'red',
}));

describe('app | components | Avatar | index', () => {
  describe('render Avatar', () => {
    it('should render Avatar', () => {
      const Component = () => <div />;

      const wrapper = mount(<Component />);

      expect(wrapper).toMatchSnapshot();
    });
  });
});
