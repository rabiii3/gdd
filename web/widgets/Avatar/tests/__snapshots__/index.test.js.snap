// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app | components | Avatar | index render Avatar should render Avatar 1`] = `
<Component>
  <div />
</Component>
`;
